mlflow/utils/databricks_utils.py

```diff
@@ -260,6 +260,19 @@ def get_job_type():
     except Exception:
         return _get_context_tag("jobTaskType")
 
+@_use_repl_context_if_available("jobType")
+def get_job_type_info():
```

Comment:
-> get_job_type()?

Reply:
Unfortunately we used that already here: https://livegrep.dev.databricks.com/view/mlflow/mlflow/mlflow/utils/databricks_utils.py#L256

I could replace all the usage but unfortunately it is used in databricks code as well: https://livegrep.dev.databricks.com/view/databricks/universe/feature-store/client/python/databricks/feature_store/utils/request_context.py#L111

So we can't swap it without a breaking change, unless I am missing something.

Reply:
I see. This is unfortunately. Do you mind filing a follow up task to clean this up. Basically get_job_type() -> get_job_task_type(), and get_job_type_info() -> get_job_type(), to be done through a slow motion across several quarters.
It's not necessary for the default experiments in job project. We could do it as a long term fixit project instead.

Reply:
Sure